odoo/addons/openupgrade_records/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+# Copyright 2011-2015 Therp BV <https://therp.nl>
+# Copyright 2016 Opener B.V. <https://opener.am>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'OpenUpgrade Records',
+    'version': '11.0.1.0.0',
+    'category': 'Migration',
+    'author': 'Therp BV, Opener B.V., OCA (Odoo Community Association)',
```

Comment:
I miss here Tecnativa

Reply:
Well, then you should have added it when you contributed to the openupgrade_records module ;-). I can add you if you point out the contribution that you made to it.